### ISSUE TITLE
Caching issue when using multiple `from_values` calls

### ISSUE THREAD

**Issue by TomFoyster** [zzzz465/timeflake4js#1]
Hi - we've found an issue with the get caching when using multiple `from_values` calls. Considering the following example;

```
const flake1 = from_values(new BN(123), new BN(456));
console.log(flake1.hex);

const flake2 = from_values(new BN(987), new BN(654));
console.log(flake2.hex);
```

**Expected behaviour**;

We would expect `flake1.hex` to log `00000000007b000000000000000001c8` and `flake2.hex` to log `0000000003db0000000000000000028e`

**Current Behaviour**;

We see both `flake1.hex` and `flake2.hex` log `00000000007b000000000000000001c8`

[Here is a test in a fork to reproduce the issue](https://github.com/TomFoyster/timeflake4js/blob/cache-issue/src/test/cache.test.ts)

It seems as though, even though `from_values` returns a new instance of `Timeflake` - the cache lib in each Timeflake object is always the same instance.

**Comment by zzzz465** [comment]
hi! thank you for submitting an issue!
I fixed the reported issue and released `v0.1.1`.

**Comment by TomFoyster** [comment]
Amazing @zzzz465 - thank you!
